umh-core/pkg/communicator/pkg/encoding/new/encoding.go

```diff
@@ -50,7 +50,8 @@ var (
 	// Add a new buffer pool for base64 operations
 	base64BufferPool = sync.Pool{
 		New: func() interface{} {
-			return make([]byte, 0, 1024) // Pre-allocate with reasonable size
```

Comment:
Im not sure what the reason behind this change is.

Reply:
fixing golangci errors